inductiva/simulators/simulator.py

```diff
@@ -54,6 +55,12 @@ def _setup_input_dir(self, input_dir: types.Path):
                 f"The provided path (\"{input_dir}\") is not a directory.")
         return input_dir
 
+    def _commands_to_json(self, cmds):
+        return [
+            cmd.to_dict() if isinstance(cmd, commands.Command) else
+            commands.Command(cmd).to_dict() for cmd in cmds
```

Comment:
keep the if/else statement in the same line (for easier reading):
```suggestion
            (cmd if isinstance(cmd, Command) else Command(cmd)).to_dict()
            for cmd in cmds
```

Also, we can import directly the Command class and reduce the line length.

Reply:
We are using the google style guide lines which are [against importing classes](https://google.github.io/styleguide/pyguide.html#22-imports). Keeping the if in the same line is completely controlled by yapf. I have no say in the matter